### DISCUSSION TITLE
Welcome to ticket-bot Discussions!

### DISCUSSION THREAD

**Discussion by Sayrix** [Sayrix/Ticket-Bot#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lolitolen** [comment]
Hi I don't found to put my token bot

**Reply by Sayrix** [reply]
Create a token.json dans put {"token": "YOURTOKEN"}

**Comment by bachyz** [comment]
Hello can someone help me with the bot?
